### ISSUE TITLE
html warnings and accessibility issues

### ISSUE THREAD

**Issue by keiranjprice101** [ISISScientificComputing/autoreduce#759]
Issue raised by: [developer]

### What?
There are lots of warnings and accessibility issues across the templates. These include missing `label` for inputs and selects, missing `alts`, `langs` and `titles`. And more general html warnings. It would be worth running an inspection in pycharm or similar to quickly identify and fix these warnings. There are also missing ARIA labels.

### Where?
These are spread throughout the templates in `WebApp/autoreduce_webapp/templates`

### How?
How did the issue come about

### Reproducible?
[Yes]

### How to test the issue is resolved
No longer have warnings and resolve some accessibility issues. Use the [Wave tool](https://wave.webaim.org/) to html and accessibility issues.

**Comment by rickybassom** [comment]
I am basing my findings around the reports generated using [Google Lighthouse](https://developers.google.com/web/tools/lighthouse/)
